src/components/wallets/AddWalletToCustomerDialog.tsx

```diff
@@ -21,16 +17,25 @@ gql`
       id
     }
   }
+
+  fragment SubscriptionForWalletCreation on Subscription {
```

Comment:
I don't understand the use of this fragment ?

Reply:
I need the `amountCurrency` from the `plan` in this component, passed in props named `userCurrency`

Reply:
Yes but `userCurrency` is a string, you don't pass the wallet, so no need for a fragment here

Reply:
indeed ! Code updated